ArduCopter/mode_brake.cpp

```diff
@@ -9,17 +9,18 @@
 // brake_init - initialise brake controller
 bool ModeBrake::init(bool ignore_checks)
 {
-    // set target to current position
-    init_target();
+    // initialise pos controller speed and acceleration
+    pos_control->set_max_speed_accel_xy(inertial_nav.get_velocity().length(), BRAKE_MODE_DECEL_RATE);
+
+    // initialise position controller
+    pos_control->init_xy_controller();
```

Comment:
We may not need to init the xy position controller each time but this is the safe option because there are a few different ways we work with the horizontal axis while the z axis is more consistent.

As this consolidates I will be looking to see if I should or should not init every time.